app/models/pseud.rb

```diff
@@ -419,6 +419,7 @@ def delete_icon
 
   def clear_icon
     self.icon = nil if delete_icon? && !icon.dirty?
+    self.icon_alt_text = nil if delete_icon?
```

Comment:
Would it make sense to extract `delete_icon?` to a guard instead, since it's required for both of these?
```suggestion
    return unless delete_icon?

    self.icon = nil if delete_icon?
    self.icon_alt_text = nil
```

Reply:
You're right, this does make more sense. Updated.